### DISCUSSION TITLE
Can't create storage buckets in local development

### DISCUSSION THREAD

**Discussion by tholder** [supabase/supabase#15023] [Questions] [answered]
Have restarted docker and run:

```
supabase stop
supabase start
```

I get an internal server error on the buckets page. The endpoint causing this is http://localhost:54323/api/storage/default/buckets

![CleanShot 2023-06-13 at 15 48 29](https://github.com/supabase/supabase/assets/41235/756c1eaf-2e82-47ad-af55-0b7e1538d2db)

Trying to create a bucket I get: "new row violates row-level security policy"

![CleanShot 2023-06-13 at 15 51 43](https://github.com/supabase/supabase/assets/41235/529969da-f67a-4c1b-a5c2-338d1ff2ca5c)

This is a clean database, none of my own tables or migrations. I've also tried to connect to db and disable RLS for all storage schema tables and no luck.

This has worked previously for me locally.

Anyone any idea what is going on?

**Comment by inian** [comment]
What version of the CLI are you using?

**Reply by tholder** [reply]
1.50.8

**Reply by tholder** [reply, marked as answer]
Sorry, just realised I had installed via brew and when I was upgrading I hadn't run brew update so it was stuck on 1.50.8.

I deleted all my containers and images, did a brew update and brew upgrade supabase. I'n now on 1.68.6 and it works fine.
